src/modelbench/benchmarks.py

```diff
@@ -57,7 +56,11 @@ def name(cls):
         return casefy.titlecase(cls.__name__.replace(BenchmarkDefinition.__name__, ""))
 
     def path_name(self):
```

Comment:
Is path_name still used anymore? If not, could we get rid of it:)

Reply:
I'll check.

Reply:
Yes, still being used by the static site generator.

Reply:
Ah ok